docs/js/search.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    function displaySearchResults(results, store) {
+      var searchResults = document.getElementById('search-results');
+      var baseURL = "/codewind";
+  
+      if (results.length) { // Are there any results?
+        var appendString = '';
+  
+        for (var i = 0; i < results.length; i++) {  // Iterate over the results
+          var item = store[results[i].ref];
+          appendString += `
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title">${item.title}</h5>
+              <p class="card-text">${item.content.substring(0, 150)}</p>
+              <a href="${baseURL}${item.url}" class="btn btn-primary">Read More</a>
+            </div>
+            <div class="card-footer">
+              <small class="text-muted">${item.layout[0].toUpperCase() + item.layout.substring(1)}</small>
+            </div>
+          </div>`;
+          // appendString += '<li class="list-group-item"><a href="' + baseURL + item.url + '"><h3>' + item.title + '</h3></a>';
+          // appendString += '<p>' + item.content.substring(0, 150) + '...</p></li>';
+        }

```

Comment:
Whichever design we go with for the cards on search, it would make sense to me to be consistent with “blogs” and “guides”... so if design prefer this one we can modify those. Personally would like the types to be a bit more prominent (maybe as a header or subtype).